less/global.less

```diff
@@ -30,9 +51,35 @@ p {
   line-height: 1.4;
 }
 
+img {
+  width: 100%;
+}
+
 .container {
   max-width: 800px;
   width: 100%;
-  margin: 0 auto;  
+  margin: 0 auto;
+}
+
+.toppart {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-items: center;
+
+  @media @phone {
+    justify-items: center;
+    background: gainsboro;
+  }
+
 }
 
+.firstt {
```

Comment:
Look into this article that talks about [CSS classes naming convention](http://bdavidxyz.com/blog/how-to-name-css-classes/)